src/components/panel/panel.js

```diff
@@ -1288,6 +1305,31 @@ MdPanelRef.prototype._configureClickOutsideToClose = function() {
 
 
 /**
+ * Configures the listeners for updating the panel position on scroll.
+ * @private
+*/
+MdPanelRef.prototype._configureScrollListener = function() {
+  var updatePosition = angular.bind(this, this._updatePosition);
+  var debouncedUpdatePosition = this._$$rAF.throttle(updatePosition);
```

Comment:
`throttle` is doing the debounce, once per animation frame. It uses the last event that happened before the frame.